app/src/main/java/it/netknights/piauthenticator/MainActivity.java

```diff
@@ -284,11 +282,11 @@ public boolean onActionItemClicked(final ActionMode mode, MenuItem item) {
         final Token currToken = tokenlistadapter.getCurrentSelection();
         final int id = item.getItemId();
         if (id == R.id.delete_token2) {
-          /*  if (currToken.isPersistent()) {
+            if (currToken.isUndeletable()) {
```

Comment:
To my understanding we show a warning message, that the token can not be deleted, if the user tries to delete the token.
Is it possible to gray out the delete button in the first place?